examples/data-objects/multiview/container/package.json

```diff
@@ -11,14 +11,12 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"main": "lib/index.js",
-	"module": "lib/index.js",
-	"types": "lib/index.d.ts",
+	"type": "module",
 	"scripts": {
```

Comment:
This one looks like it is meant to be library+app and thus should have exports. I think the indicator is src/index.ts with something like `export const fluidExport`. @ChumpChief may know more.

Reply:
Update: pattern has changed but stale things like these package entrypoints weren't updated. New pattern is to use webpacked result. (Not sure what to check for to make sure webpack'ed library is sound. But that can be another day's concern and at least multiview-container is private package.)